docs/guide.mdx

```diff
@@ -73,6 +73,12 @@ wave.strategy = ['dockerfile']
 wave.build.repository = 'docker.io/<user>/<repository>'
 ```
 
+This would build a new image, push it to docker.io and return a URI with the following structure:
+
+```
+docker.io/<user>/<repository>:<image>--<hash>
+```
```

Comment:
````suggestion
```

The image `tag` here is composed of a Human-readable `<image>` name and a `<hash>` based on the build instructions.
````